airflow/example_dags/example_subdag_operator.py

```diff
@@ -19,29 +19,29 @@
 """Example DAG demonstrating the usage of the SubDagOperator."""
 
 # [START example_subdag_operator]
+from datetime import datetime
+
 from airflow import DAG
 from airflow.example_dags.subdags.subdag import subdag
 from airflow.operators.dummy import DummyOperator
 from airflow.operators.subdag import SubDagOperator
-from airflow.utils.dates import days_ago
 
 DAG_NAME = 'example_subdag_operator'
 
-args = {
```

Comment:
Update this to a more relevant example/use of `default_args` other than `owner`. (Some could argue setting `owner` _might_ be irrelevant with RBAC.)